# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1550510141 0
#      Mon Feb 18 17:15:41 2019 +0000
# Node ID 1d2552e895e6ba6e2613a861c3dc267a4dada95a
# Parent  9cadeae949f2c4a2099e2a391a4da548e146019b
* scripts/miscellaneous/module.mk: Fix file name added in previous changeset.

diff --git a/scripts/miscellaneous/module.mk b/scripts/miscellaneous/module.mk
--- a/scripts/miscellaneous/module.mk
+++ b/scripts/miscellaneous/module.mk
@@ -31,17 +31,17 @@ FCN_FILE_DIRS += \
   %reldir%/gunzip.m \
   %reldir%/info.m \
   %reldir%/inputParser.m \
   %reldir%/inputname.m \
   %reldir%/isdeployed.m \
   %reldir%/isfile.m \
   %reldir%/isfolder.m \
   %reldir%/ismac.m \
-  %reldir%/ismember.m \
+  %reldir%/ismethod.m \
   %reldir%/ispc.m \
   %reldir%/isunix.m \
   %reldir%/license.m \
   %reldir%/list_primes.m \
   %reldir%/loadobj.m \
   %reldir%/ls.m \
   %reldir%/ls_command.m \
   %reldir%/menu.m \
